### DISCUSSION TITLE
Snapshot button?

### DISCUSSION THREAD

**Discussion by jodewee** [dougiteixeira/proxmoxve#198] [Feature request] [answered]
Would it be possible to add a button to take a snapshot of a ct or vm?

**Comment by mZ738** [comment, marked as answer]
A `Create snapshot` button exists per VM and container since 5.1.0 ([release notes](https://github.com/dougiteixeira/proxmoxve/releases/tag/5.1.0)): a disk-only snapshot named `homeassistant_<date>_<time>`, described as created by Home Assistant, needing `VM.Snapshot`. Off by default like the other buttons that change things; it appears only where the credentials hold the privilege. Backups came along with it — buttons, a `proxmoxve.backup` action and a scheduled-backup blueprint, see [Actions](https://github.com/dougiteixeira/proxmoxve/blob/main/docs/actions.md). Closing as done.
